backend-lucene/src/main/java/org/hibernate/search/v6poc/backend/lucene/document/model/dsl/impl/LuceneIndexSchemaElementImpl.java

```diff
@@ -72,4 +79,13 @@ public LuceneIndexSchemaObjectField objectField(String relativeFieldName, Object
 					return new LuceneIndexSchemaObjectFieldImpl( nodeBuilder, filter );
 				} );
 	}
+
+	private static void checkRelativeFieldName(String relativeFieldName) {
+		if ( relativeFieldName == null || relativeFieldName.isEmpty() ) {
+			throw log.relativeFieldNameCannotBeNullOrEmpty( relativeFieldName );
+		}
+		if ( relativeFieldName.contains( "." ) ) {
+			throw log.relativeFieldNameCannotContainDot( relativeFieldName );
+		}
```

Comment:
Obviously exactly the same comments for the Lucene part.

Reply:
Same answer